sycl/doc/extensions/proposed/sycl_ext_intel_fpga_datapath.asciidoc

```diff
@@ -212,24 +264,44 @@ Returns a reference to the underlying `T`.
 
 Below are some non-obvious interactions between `fpga_datapath` and other SYCL extensions. 
 
-* The `fpga_datapath` and `fpga_mem` are tempate classes that have conflicting requirements on how a variable should be implemented, on or off the datapath. In situations where an object of one of these classes is templated on another, the most specific template will used by the compiler, and the rest will be ignored.
+* Since fpga_mem/fpga_datapath applies to the entire object that it wraps, conflicting hints may be applied the same object, such as in the example below.
```

Comment:
```suggestion
* Since fpga_mem/fpga_datapath applies to the entire object that it wraps, conflicting hints may be applied to the same object, such as in the example below.
```